docker/ethereum/ethereum.go

```diff
@@ -0,0 +1,119 @@
+package ethereum
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/config/types"
+	docker_utils "github.com/smartcontractkit/chainlink-testing-framework/utils/docker"
+)
+
+type Fork string
+
+const (
+	EthereumFork_Shanghai Fork = "Shanghai"
+	EthereumFork_Deneb    Fork = "Deneb"
+)
+
+func (e Fork) GetValidFutureForks() ([]Fork, error) {
```

Comment:
[Avoid using Get in function names](https://google.github.io/styleguide/go/best-practices.html#naming-conventions) + comment please